lite2/client.go

```diff
@@ -62,13 +62,14 @@ func UpdatePeriod(d time.Duration) Option {
 	}
 }
 
-// RemoveNoLongerTrustedHeadersPeriod option can be used to define how often
-// the routine, which cleans up no longer trusted headers (outside of trusting
-// period), is run. Default: once a day. When set to zero, the routine won't be
-// started.
-func RemoveNoLongerTrustedHeadersPeriod(d time.Duration) Option {
+// PruningSize option sets the maximum amount of headers and validator sets that
+// the light client stores. When Prune() is run, all headers that are earlier than
+// the h amount of headers will be removed from the store. If UpdatePeriod is used,
```

Comment:
I don't think we should combine these two. I mean it totally makes sense to have pruning when you have auto update (default should be: pruning + auto update), but maybe leave the option to not have pruning even if auto update is enabled

Reply:
That's possible if you have a pruning size of 0 and an updatePeriod > 0